config/containers/logging/configure.md

```diff
@@ -135,24 +135,22 @@ Docker provides two modes for delivering messages from the container to the log
 driver:
 
 * (default) direct, blocking delivery from container to driver
-* non-blocking delivery that stores log messages in an intermediate per-container
-  ring buffer for consumption by driver
+* non-blocking delivery that stores log messages in an intermediate per-container buffer for consumption by driver
```

Comment:
I'm guessing this edit will be contentious and I can remove it if you want. But with the way the code actually works it is not a true ring buffer, its just a buffer: https://github.com/moby/moby/blob/master/daemon/logger/ring.go#L160

Reply:
Nah, this is perfect. It describes reality. I don't recall how these docs got written but they are definitely wrong.